lib/internal/module.js

```diff
@@ -50,6 +51,40 @@ function stripBOM(content) {
   return content;
 }
 
+/**
+ * Find end of shebang line and slice it off
+ */
+function stripShebang(content) {
+  // Remove shebang
+  var contLen = content.length;
+  if (contLen >= 2) {
+    if (content.charCodeAt(0) === 35/*#*/ &&
+        content.charCodeAt(1) === 33/*!*/) {
+      if (contLen === 2) {
+        // Exact match
+        content = '';
+      } else {
```

Comment:
You could just drop the `if (contLen === 2)` case, it should almost never be hit.